uiautomationutilities/build.gradle

```diff
@@ -42,10 +42,14 @@ android {
 
     def final installSourcePlayStore = "PlayStore"
     def final installSourceLocalApk = "LocalApk"
+    def final updateSourcePlayStore = "PlayStore"
+    def final updateSourceLocalApk = "LocalApk"
```

Comment:
There is duplication of constants now. I think you can reduce these to just 2 as opposed to 4:

- appSourcePlayStore = "PlayStore"
- appSourceLocalApk = "LocalApk"